lib/config/options/index.ts

```diff
@@ -1272,6 +1272,18 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'matchReleaseAge',
+    description:
+      'Matches the age of a proposed package update. Renovate uses the package release timestamp to decide the package age. Valid only within a `packageRules` object.',
```

Comment:
The `description` for `matchCurrentAge` says:

> Matches the current age of the package derived from its release timestamp. Valid only within a `packageRules` object.`

We can use a similar format for the new feature:

```suggestion
      'Matches the release age of the package derived from its release timestamp. Valid only within a `packageRules` object.',
```

Reply:
```suggestion
      'Matches the age of a proposed package update derived from its release timestamp. Valid only within a `packageRules` object.',
```

This is better as it specifies that this is the age of the release which will be proposed in an update PR.
Only, saying `release age of the package` is incomplete in this case.

Reply:
Let's go with `@RahulGautamSingh`'s version then. 😉